node_package/src/buildConsoleReplay.ts

```diff
@@ -1,6 +1,8 @@
 import RenderUtils from './RenderUtils';
 import scriptSanitizedVal from './scriptSanitizedVal';
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
```

Comment:
This part needs to be improved. I couldn't find a way other than ignoring the linter rule.